plugins/azure-functions/src/api/types.ts

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2022 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/** @public */
+export type FunctionsData = {
```

Comment:
This might feel like overkill but it might be worth creating a `azure-functions-common` package that has shared code like these types. Right now you just have this `FunctionsData` but if this grows, and I suspect it will, you'll be happy to have this in place early on

Reply:
Good suggestion. I have created a new common-library through the cli